ext/nmatrix/math/util.h

```diff
@@ -145,4 +145,24 @@ static inline char lapack_evd_job_sym(VALUE op) {
   else return 'V';
 }
 
+/*
+ * Interprets the side and trans arguments for LAPACK function lapack_ormqr/lapacke_unmqr
+ *
+ * Related to obtaining Q in QR factorizationa after calling lapack_geqrf
+ */
+
+static inline char lapack_qr_side(VALUE op) {
```

Comment:
The symbols accepted by this function should probably match those used `blas_side_sym` above. Also, rename this to `lapacke_side_sym`